fw-child/template/social-networks.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+//
+// Display social network links.
+//
+$current_lang = 'en';
```

Comment:
Just to be consistent with file descriptions, I would put this description as a multiline comments (and an empty line after). Example:

```suggestion
/**
 * Display social network links.
 */

$current_lang = 'en';
```